protocol/daemons/slinky/client/client.go

```diff
@@ -17,25 +18,28 @@ import (
 	libtime "github.com/dydxprotocol/v4-chain/protocol/lib/time"
 )
 
-// Ensure Client is HealthCheckable
-var _ daemontypes.HealthCheckable = (*Client)(nil)
-
 // Client is the daemon implementation for pulling price data from the slinky sidecar.
 type Client struct {
-	daemontypes.HealthCheckable
```

Comment:
does anything depend on this `Client` having the embedded `HealthCheckable` type?

if so, we could update HealthCheckable to track multiple success/error states within it

but this might not necessary, so asking the question!

Reply:
I don't believe so, right now there isn't anything to allow `HealthCheckable` to aggregate across two `HealthCheckable`s